RIck and Morty/ViewModel/ListViewModel.swift

```diff
@@ -9,21 +9,29 @@
 import Foundation
 
 class ListViewModel {
-  
-  var networkManager = NetworkManager()
-  var characterList: [CharacterModel] = [CharacterModel]()
-  var currentPage = 1
-  var currentCount: Int {
-    return characterList.count
+
+  private (set) var dataSource: [CharacterCellViewState] = []
```

Comment:
see how i changed the visibility of every property and I converted some of them to inmutable constants